File: SEP3 - TIER 3 -DBS/src/Data/Absence.java
package Data;

public class Absence
{
    private String date;
    private boolean motivated;
    private String course;

    public Absence(String date, boolean motivated, String course)
    {
        this.date = date;
        this.motivated = motivated;
        this.course = course;
    }

    public String getDate()
    {
        return date;
    }

    public void setDate(String date)
    {
        this.date = date;
    }

    public boolean isMotivated()
    {
        return motivated;
    }

    public void setMotivated(boolean motivated)
    {
        this.motivated = motivated;
    }

    public String getCourse()
    {
        return course;
    }

    public void setCourse(String course)
    {
        this.course = course;
    }
}

File: SEP3 - TIER1/Gradebook/Models/Absence.cs
namespace WebApplication.Models
{
    public class Absence
    {
        private string date;
        private bool motivated;
        private string course;

        public Absence(string date, bool motivated, string course)
        {
            this.date = date;
            this.motivated = motivated;
            this.course = course;
        }

        public string getDate()
        {
            return date;
        }

        public void setDate(string date)
        {
            this.date = date;
        }

        public bool isMotivated()
        {
            return motivated;
        }

        public void setMotivated(bool motivated)
        {
            this.motivated = motivated;
        }

        public string getCourse()
        {
            return course;
        }

        public void setCourse(string course)
        {
            this.course = course;
        }
    }
}